### ISSUE TITLE
support amp-iframe

### ISSUE THREAD

**Issue by JoeGermuska** [juusaw/amp-jekyll#30]
I understand that this is not supported yet, but noting it as a feature request. 

Are you interested in pull requests for other AMP tags such as this?

**Comment by juusaw** [comment]
Definitely! I'll happily accept pull requests for all tag implementations.
